docs/installation_for_mac.md

```diff
@@ -0,0 +1,80 @@
+
+# Installation for Mac ARM64
+
+When it comes to installing OasisLMF on Mac ARM64, the pre-complied binaries are not available. Therefore, the 
+installation process is a bit more complicated and the compilation needs to be local. The following steps are required 
+to install OasisLMF on Mac ARM64:
+
+First we clone the [ktools](https://github.com/OasisLMF/ktools) repo with the following command:
+
+```bash
+git clone https://github.com/OasisLMF/ktools.git
+```
+
+Before we run anything, we need to ensure that the following dependencies are installed:
+
+```bash
+brew install \
+    autoconf \
+    automake \
+    libtool \
+    zlib-ng
+```
+
+We then move into the ktools directory and run the following commands:
+
+```bash
+./autogen.sh
+```
+
+We then confgirue the ktools with the following command:
+
+```bash
+./configure --enable-osx --enable-o3 --prefix=<BIN_PATH>
```

Comment:
can we say what the user should use for BIN_PATH? I guess something like `$HOME/ktools/bin`, if the user cloned ktools into $HOME?